pandas/io/excel/_util.py

```diff
@@ -21,16 +16,11 @@ def register_writer(klass):
         raise ValueError("Can only register callables as engines")
     engine_name = klass.engine
     _writers[engine_name] = klass
-    for ext in klass.supported_extensions:
-        if ext.startswith('.'):
-            ext = ext[1:]
-        if ext not in _writer_extensions:
-            config.register_option("io.excel.{ext}.writer".format(ext=ext),
-                                   engine_name, validator=str)
-            _writer_extensions.append(ext)
 
 
 def _get_default_writer(ext):
+    """Return the default writer per extension. This default engine is used
```

Comment:
Can you follow pandas docstring conventions here? You can run the `scripts/validate_docstrings.py` script against this to get a list of errors